src/Nucleation.jl

```diff
@@ -0,0 +1,97 @@
+module Nucleation
+
+"""
+    nucleation_timestep(rh, temp, so4)
+
+Calculates the nucleation rate for a single timestep under the given conditions.
+All newly formed particles will be placed into the Aitken mode.
+
+The initial nucleation rate comes from Vehkamaki et al, 2002 doi:10.1029/2002JD002184
+Adjustment factor comes from Kerminen and Kulmala, 2002 doi:10.1029/2002JD002184
+
+# Inputs
+rh: Relative humidity, expressed as a percentage between 0 and 1
+temp: Temperature (K)
+so4: Total gas phase concentration of sulfuric acid (cm^-3)
+
+# Examples
+```julia-repl
+julia> nucleation_timestep(0.55, 236, 1e7)
+162.96173344575243
+```
+"""
+
```

Comment:
```suggestion
```
I think doc strings cannot have newlines separating the functions / types

Reply:
@charleskawczynski Sorry, I'm a little confused by the suggestion. How's this?
```
"""
    nucleation_timestep(rh, temp, so4)

 - `rh` - Relative humidity, expressed as a percentage between 0 and 1
 - `temp` - Temperature (K)
 - `so4` - Total gas phase concentration of sulfuric acid (cm^-3)

Returns the nucleation rate for a single timestep under the given conditions.
All newly formed particles will be placed into the Aitken mode.
The initial nucleation rate comes from Vehkamaki et al, 2002 doi:10.1029/2002JD002184
Adjustment factor comes from Kerminen and Kulmala, 2002 doi:10.1029/2002JD002184

# Examples
```julia-repl
julia> nucleation_timestep(0.55, 236, 1e7)
162.96173344575243
```
I copied the format from [remove_precipitation](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/src/Microphysics0M.jl#L21).

One more thing - the PR doesn't include any tests, but I have a plotting script to make sure that it's correct. What should I do about codecov failing?

Reply:
Hi @nefrathenrici, the general format is fine. The difference is that [here](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/src/Microphysics0M.jl#L35-L36), there is no line between the function and the doc string. For example

##### Broken doc string:
```julia
"""
    Foo

"""

Foo() = 1
```

##### Working doc string:
```julia
"""
    Foo

"""
Foo() = 1
```

Reply:
> One more thing - the PR doesn't include any tests, but I have a plotting script to make sure that it's correct. What should I do about codecov failing?

I think there's two things here:

 - First, our CI doesn't depend on CodeCov-- it's more of a warning signal if, for example, coverage suddenly drops 90%. The reason being that CodeCov is [not 100% accurate](https://github.com/JuliaCI/Coverage.jl#a-note-for-advanced-users) due to how it's tracked and compiler optimizations.
 - Second, the plotting script should probably be moved to the `test/` or `docs/`, so that we don't have to make CloudMicrophysics.jl depend on (and therefore ship with) Plots.jl, since this is a pretty heavy-weight package. The other advantage of this is that, if we add it to the docs, which I believe already has plots, is that we can run this script as a part of CloudMicrophysics.jl's CI, to ensure that it consistently works as changes are made

Reply:
OK, should be fixed now. Thanks for clarifying everything!